README.md

```
@@ -65,136 +57,311 @@ counters. Follow this
-at container startup time.
+##### vLLM
 
-> **_NOTE_**: if you do provide a triton_path you should run `git submodule init`
-and `git submodule update` on the mounted repo if you haven't already run these
-commands.
+```sh
+ make vllm-cuda-run [OPTIONS]
+```
 
-> **_NOTE_**: the `user_path` will be mounted inside the container at `/workspace/user`.
+#### CPU container
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)